openpype/hosts/tvpaint/plugins/publish/extract_sequence.py

```diff
@@ -23,12 +24,14 @@
 
 
 class ExtractSequence(pyblish.api.Extractor):
+    order = pyblish.api.ExtractorOrder + 0.01
```

Comment:
Why is this needed? Wouldn't it make more sense to just move the ExtractJson to `-0.01`?